src/lib.rs

```diff
@@ -446,6 +446,84 @@ where
     }
 }
 
+impl<S> WebSocketStream<S> {
+    /// Simple send method to replace `futures_sink::Sink` (till v0.3).
+    pub async fn send(&mut self, msg: Message) -> Result<(), WsError>
+    where
```

Comment:
GitHub doesn't update this properly for comments, but in the diff view, it's not controlled by feature flag now

Reply:
Yeah, GHs PR review UI is quite horrible so I went through the whole changes again. They should learn from GitLab ;)

Thanks for pointing that out